zebra/debug.c

```diff
@@ -189,6 +192,17 @@ DEFUN (debug_zebra_vxlan,
 	return CMD_SUCCESS;
 }
 
+DEFUN (debug_zebra_gre,
```

Comment:
let's prefer DEFPY now, and use a single stanza for both the positive and negative forms - see debug_zebra_evpn_mh or debug_zebra_nexthop

Reply:
ok